SubscribeIT/ngDesk
# User name logo bug

Closes #327 

Test Case 1
Name :-  Chat edit layout 
1. Login to ngdesk.
2. Setup chat widget.
3. Accept chats.
4. Goto -> chats -> click on active chat.
5. Goto -> chat widget.
6. Type some message.
7. Check the chat page.
8. Observe no error.

Test Case 2
Name :-  Chat edit layout with update program
1. Login to ngdesk.
2. Post company and run update program.
3. Setup chat widget.
4. Accept chats.
5. Goto -> chats -> click on active chat.
6. Goto -> chat widget.
7. Type some message.
8. Check the chat page.
9. Observe no error.

Screenshots :- 

![Screenshot from 2022-02-03 09-13-56](https://user-images.githubusercontent.com/89504119/152277853-58fb4e12-a71f-4b23-a29f-cf99f5a58651.png)
